src/data-management/Utils/get-accessmode-from-str.ts

```diff
@@ -0,0 +1,32 @@
+import { AccessMode } from "../data-model/authorization/access-mode";
+
+export function getAccessmodeFromStr(accessMode: string): AccessMode {
+  let accessModeEnum: AccessMode;
+  switch (accessMode) {
+    case "http://www.w3.org/ns/auth/acl#Read": {
```

Comment:
I stedet for at skrive hele urlen så lav en variable til det der er det samme som eventuelt hedder baseUrl som du så bruger til at sætte sammen med resten. Fx: baseUrl = "http://www.w3.org/ns/auth/"
case baseUrl + "acl#Read"